### PR TITLE
add steps to update galaxy.yml in publish action

### PR DESCRIPTION
**Pull request by rndmh3ro**

### CODE REVIEW

File: .github/workflows/galaxy.yml [hunk 1]
```diff
@@ -11,8 +11,28 @@ jobs:
...
         galaxy_version: ${{ github.event.release.tag_name }}
+
+    - name: update galaxy.yml with new version
+      uses: microsoft/variable-substitution@v1
+      with:
+        files: 'galaxy.yml'
+      env:
+        version: "${{ github.event.release.tag_name }}"
+
+    - name: push changelog
```
Comment (schurzi) [inline .github/workflows/galaxy.yml:29]: the name should be updated to reflect the file we are dealing with
```suggestion
    - name: push galaxy.yml
```

**Comment by schurzi** [comment]
I have problems understanding this, won't we need to bump the version in galaxy.yml before pushing the packacke to galaxy?

**Comment by rndmh3ro** [comment]
No, we don't need this because the github-action does this dynamically. See here: https://github.com/marketplace/actions/deploy-ansible-galaxy-collection

**Comment by schurzi** [comment]
oh nice, I see, it takes the GitHub release tag for this. In this case, do we even need to track the version in `galaxy.yml`?

**Comment by rndmh3ro** [comment]
Yes, that's required by ansible.

**Pull request merged**